Lib/test/test_fstring.py

```diff
@@ -940,15 +940,13 @@ def test_lambda(self):
                              "f'{lambda :x}'",
                              "f'{lambda *arg, :x}'",
                              "f'{1, lambda:x}'",
+                             "f'{lambda x:}'",
+                             "f'{lambda :}'",
                              ])
 
         # but don't emit the paren warning in general cases
-        self.assertAllRaise(SyntaxError,
-                            "f-string: expecting a valid expression after '{'",
-                            ["f'{lambda x:}'",
-                             "f'{lambda :}'",
-                             "f'{+ lambda:None}'",
```

Comment:
Why are we removing the other cases?

Reply:
Just moved them up to the check for the specialized error message, since these now raise the "parenthesize your lambdas" error.